src/components/link/link.svelte

```diff
@@ -40,7 +40,7 @@
     --focus-shadow: var(--leo-link-focus-shadow, var(--leo-effect-focus-state));
 
     color: var(--color);
-    font: var(--leo-font-primary-default-semibold);
+    font: var(--leo-link-font, inherit);
```

Comment:
We can probably just completely remove this

Reply:
```suggestion
```